### PR TITLE
Fix footer padding

### PR DESCRIPTION
**Pull request by alicoding**
@ScottDowne can you r? please

### CODE REVIEW

File: call-congress.css [hunk 1]
```diff
@@ -482,6 +484,8 @@ and (max-width : 480px) {
   .branding {
     float: left;
     width: 100%;
+    padding-bottom: 45px;
+    height: 30px;
```
Comment (ScottDowne) [inline call-congress.css]: height looks to be overriding padding-bottom here, I think you can remove these two if you also remove padding-bottom: 45px on line 500

**Comment by ScottDowne** [comment]
Height 480-780ish doesn't seem to have any bottom padding in the footer.

**Comment by ScottDowne** [comment]
Hm, that last comment isn't fully correct, it has a padding bottom, but seem the height is wrong so the padding bottom doesn't do enough? Not sure.

**Comment by alicoding** [comment]
r+ed by Scott on IRC

**Pull request merged**